main/BUILD

```diff
@@ -25,7 +34,8 @@ cc_binary(
         "//conditions:default": 1,
     }),
     malloc = select({
-        ":jemalloc": "@jemalloc//:jemalloc",
+        # ":jemalloc": "@jemalloc//:jemalloc",
```

Comment:
Should we have both of these, so that you can use the config setting to pick whichever one you prefer from the bazel build command line?